semconv/http.go

```diff
@@ -26,8 +26,8 @@ import (
 )
 
 var (
-	HTTPSchemeHTTP  = HTTPSchemeKey.String("http")
-	HTTPSchemeHTTPS = HTTPSchemeKey.String("https")
+	httpSchemeHTTP  = HTTPSchemeKey.String("http")
+	httpSchemeHTTPS = HTTPSchemeKey.String("https")
```

Comment:
This is a breaking change.
I guess these might have been intentionally exported.